prime-angular-frontend/src/app/modules/enrolment/pages/pharmanet-enrolment-summary/pharmanet-enrolment-summary.component.ts

```diff
@@ -189,37 +190,51 @@ export class PharmanetEnrolmentSummaryComponent extends BaseEnrolmentPage implem
   }
 
   public sendProvisionerAccessLink() {
-    const data: DialogOptions = {
-      title: 'Confirm Email',
-      message: `Are you sure you want to send your Approval Notification?`,
-      actionText: 'Send',
-    };
-    this.busy = this.dialog.open(ConfirmDialogComponent, { data })
-      .afterClosed()
-      .pipe(
-        exhaustMap((result: boolean) => {
-          if (result) {
-            let emailPairs = this.careSettingConfigs.map((config) => {
-              return {
-                emails: config.formArray.value.map(email => email.email),
-                careSettingCode: config.settingCode,
-                healthAuthorityCode: config.healthAuthorityCode,
-              }
-            });
-            return this.enrolmentResource.sendProvisionerAccessLink(emailPairs, this.enrolment.id);
-          } else {
-            return EMPTY;
-          }
-        })
-      )
-      .subscribe(() => {
-        let emails = new Array<string>();
-        this.careSettingConfigs.forEach((config) => {
-          emails.push(config.formArray.value.map(email => email.email));
+
+    if (!this.atLeastOneEmailFilled()) {
+      const data: DialogOptions = {
+        title: 'Missing Email',
+        message: `Please enter at least one email for the Approval Notification?`,
```

Comment:
I think question mark at end of sentence is unnecessary and icon should be "warning" or "error"

Reply:
Update as suggested.